packages/@aws-cdk/assert/package.json

```
@@ -53,7 +53,7 @@
   "peerDependencies": {
     "@aws-cdk/core": "0.0.0",
     "constructs": "^3.3.69",
-    "jest": "^26.6.3"
+    "jest": "^26.6.3 || ^27.0.6"
```

Should probably just be -

```suggestion
    "jest": ">=26.6.3"
```

---

Good point, updated